integration-cli/docker_cli_rename_test.go

```diff
@@ -97,7 +97,7 @@ func TestRenameCheckNames(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	logDone("rename - running container")
+	logDone("rename - check the first container's status")
```

Comment:
Hi cpuguy83,
How about the following?
logDone("rename - make sure that a container can't be inspected by its previous name after it is renamed.")